c++/src/kj/mutex.c++

```diff
@@ -59,6 +59,51 @@
 #endif
 
 namespace kj {
+#if KJ_TRACK_LOCK_BLOCKING
+static thread_local BlockedOnReason* tlsBlockReason __attribute((tls_model("initial-exec")));
```

Comment:
The commit message could use an update.

Reply:
Will leave this open so I can remember to do it once I squash the fixups.